R/stage_nldas_ts.R

```diff
@@ -24,7 +24,7 @@
 stage_nldas_ts <- function(sites, var, times, folder = tempdir(), verbose = FALSE, ...){
   
   if(length(var) > 1) stop("one var at a time, please")
-  p_code <- get_var_codes(var) %>% filter(src=="nldas") %>% select(p_code)
+  p_code <- get_var_codes(var) %>% filter(src=="nldas") %>% select(p_code) %>% as.character()
```

Comment:
well, shoot. sorry. i forgot that filter and select don't drop the data.frame structure.